demos/instr.h

```diff
@@ -77,4 +77,31 @@ inline void JumpToAfterSpeculation() {
   asm volatile("b afterspeculation");
 }
 #endif
+
+#if defined(__i386__) || defined(__x86_64__)
+__attribute__((always_inline))
```

Comment:
Does this really need to be inlined?

Reply:
Yes. Without that the demonstration leaks always only one character and then it waits at least for a couple of seconds.